app/html/package.json

```diff
@@ -21,11 +21,17 @@
     "prepare": "node ../../scripts/prepare.js"
   },
   "dependencies": {
+    "@babel/core": "^7.0.0-rc.1",
+    "@babel/runtime": "^7.0.0-rc.1",
     "@storybook/core": "4.0.0-alpha.16",
+    "babel-core": "^7.0.0-bridge.0",
```

Comment:
Do we really need babel in html app? I am not actually sure why we added it to all apps.